### PR TITLE
Merge MSAL dev to release/0.5.0

### PR DESCRIPTION
**Pull request by oldalton**

### CODE REVIEW

File: MSAL.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "MSAL"
-  s.version      = "0.5.0"
```
Comment (jasoncoolmax) [inline MSAL.podspec]: should this be 0.5.0? :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,14 +16,32 @@
...
+- Updated to newer v2 broker protocol version
+
 ## [0.3.1] - 2019-05-06
 ### Fixed
 - Better error handling in CBA cancellation flows
 - Don't read corrupted refresh tokens from cache
 
 ## [0.3.0] - 2019-04-22
 ### Added
+<<<<<<< HEAD
```
Comment (jasoncoolmax) [inline CHANGELOG.md]: this conflict hasn't been addressed yet :P

File: MSAL/resources/ios/Info.plist [hunk 3]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.5.0</string>
```
Comment (jasoncoolmax) [inline MSAL/resources/ios/Info.plist]: keep it 0.5.0?

File: MSAL/resources/mac/Info.plist [hunk 4]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.5.0</string>
+	<string>0.4.3</string>
 	<key>CFBundleVersion</key>
```
Comment (jasoncoolmax) [inline MSAL/resources/mac/Info.plist]: keep 0.5.0?

Comment (oldalton) [reply MSAL/resources/mac/Info.plist]: yes, fixed :)

File: MSAL/resources/ios/Info.plist (hunk 3 above)
Comment (oldalton) [reply MSAL/resources/ios/Info.plist]: yes, fixed :)

File: CHANGELOG.md (hunk 2 above)
Comment (oldalton) [reply CHANGELOG.md]: yes, fixed :)

File: MSAL.podspec (hunk 1 above)
Comment (oldalton) [reply MSAL.podspec]: yes, fixed :)

File: CHANGELOG.md [hunk 5]
```diff
@@ -20,6 +38,8 @@
 ## [0.3.0] - 2019-04-22
 ### Added
 -Added broker support to MSAL iOS SDK
+=======
```
Comment (jasoncoolmax) [inline CHANGELOG.md]: conflict not resolved here :)

**Pull request merged**